### ISSUE TITLE
Ionic: Video and audio issue

### ISSUE THREAD

**Issue by siraj1044** [muaz-khan/RecordRTC#340]
When recording video plus audio in Android using ionic, there is a lot of lag in video and audio is completely missing. I am recording remote stream in a video call, when user stop recording it is saved in android file system. 

        `var options = {
          mimeType: 'video/webm\;codecs=h264'
        };

        $scope.recordRTC = RecordRTC($scope.remoteVideoStream, options);
        $scope.recordRTC.startRecording();

        //for stoping record
        $scope.recordRTC.getDataURL(function (dataURL) {
            //save base64 data in file
        }`
      
Let me know if i am doing anything wrong

**Comment by muaz-khan** [comment]
Did you try vp9/vp8?

**Comment by siraj1044** [comment]
Yes same issue, although lagging issue is better but it still lags and no audio.

**Comment by siraj1044** [comment]
Any suggestions??

**Comment by muaz-khan** [comment]
Still an issue?

**Comment by siraj1044** [comment]
yes same issue while using vp9/vp8, video is lagging plus no audio.
